### ISSUE TITLE
solana-wallet delegate-stake crashes when the stake keypair file doesn't exist

### ISSUE THREAD

**Issue by mvines** [solana-labs/solana#5563]
```bash
$ solana-wallet delegate-stake idontexist.json ~/validator-vote-keypair.json 123
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:347:21
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by garious** [comment]
Please ensure this entire class of errors goes away (possible panic of any CLI tool that reads and parses a file).  Need generic tests for nonexistent file, empty file, garbage data, etc.

**Comment by CriesofCarrots** [comment]
The issue is that there is no `.validator()` on the delegate-stake `stake_account_keypair_file` arg, but `parse_command()` assumes there is.

**Comment by CriesofCarrots** [comment]
Fixed by #5567
